src/components/DonationTable/DonationTable.tsx

```diff
@@ -118,7 +118,9 @@ export const createDonationRow = (
   currency: data.amount.convertedCurrency,
   foreignAmount: data.amount.amount,
   foreignCurrency: data.amount.currency,
-  designationAccount: data.designationAccount.name,
+  designationAccount: data.designationAccount.name
+    ? data.designationAccount.name
+    : data.designationAccount.accountNumber,
```

Comment:
Could you also do
```suggestion
  designationAccount: data.designationAccount.name ?? data.designationAccount.accountNumber,
```

Reply:
@dr-bizz `data.designationAccount.name` returns an empty string if there is no name. So that is why I had to use `?` instead of `??`. `??` returns true for empty strings.

Reply:
Ah, try `||` instead.
```suggestion
  designationAccount: data.designationAccount.name || data.designationAccount.accountNumber,
```